litgpt/api.py

```diff
@@ -6,12 +6,12 @@
 
 import torch
 import lightning as L
-from lightning.fabric.utilities.device_parser import _normalize_parse_gpu_input_to_list
 
 from litgpt.model import GPT  # needs to be imported before config
 from litgpt.config import Config
 from litgpt.tokenizer import Tokenizer
 from litgpt.generate.base import generate as generate_fn
+from litgpt.chat.base import generate as stream_generate
```

Comment:
Minor thing, but could you change it to

```suggestion
from litgpt.chat.base import generate as stream_generate_fn
```

for consistency